host/sgx/sgxload.c

```diff
@@ -122,7 +123,23 @@ static int _make_memory_protect_param(uint64_t inflags, bool simulate)
     return outflags;
 }
 
-static sgx_secs_t* _new_secs(uint64_t base, size_t size, bool debug)
+/* Detect the XSave Feature Request Mask (XFRM) to set in the enclave */
+uint64_t _xfrm_detect()
```

Comment:
Style: function naming is always `_verb_noun()` so `_detect_xfrm()`

Reply:
ok, fixed